packages/next/src/server/lib/start-server.ts

```diff
@@ -279,11 +278,11 @@ export async function startServer(
           // This is the render worker, we keep the process alive
           console.error(err)
         }
-        process.on('exit', (code) => cleanup(code))
```

Comment:
Why we don't need to handle `'exit'` anymore?

Reply:
The `exit` event is sent right before the process exits and it can only execute synchronous code. I suppose we could capture it if we wanted to force the exit code to be `0`, but I was thinking letting it send the code it got made sense. 
https://nodejs.org/api/process.html#event-exit

> There is no way to prevent the exiting of the event loop at this point, and once all `'exit'` listeners have finished running the Node.js process will terminate.

> Listener functions **must** only perform **synchronous** operations. The Node.js process will exit immediately after calling the `'exit'` event listeners causing any additional work still queued in the event loop to be abandoned.

Since `server.close` is asychronous, I don't think calling it on the `'exit'` event would do anything useful

https://nodejs.org/api/net.html#serverclosecallback

> This function is asynchronous

I could see there maybe being a couple things that `server.close` does synchronously, but it definitely would exit without calling the callback.

Reply:
Also just realized it was already using the original exit code and just defaulting to 0, so the behavior should still be the same without listening for the event